### PR TITLE
CORE-4060: testnet replaced with devnet for harmony

### PR DESCRIPTION
**Pull request by GranderStark**
Values were replaced as part of migration from testnet to devnet.

### CODE REVIEW

File: docs/operations/harmony/tools.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ You can set your [MetaMask](https://metamask.io/) to interact through your Harmo
 1. In the **Chain ID** field, enter the ID of the network:
 
     * Mainnet Shard 0: `1666600000`
-    * Testnet Shard 0: `1666700000`
+    * Devnet Shard 0: `1666700000`
```
Comment (akegaviar) [inline docs/operations/harmony/tools.md:89]: `1666900000` see https://docs.harmony.one/home/general/wallets/browser-extensions-wallets/metamask-wallet/adding-harmony

File: docs/operations/harmony/tools.md [hunk 2]
```diff
@@ -544,7 +544,7 @@ where
 * PASSWORD — your node access password.
 * NETWORK_ID — Harmony network ID:
   * Mainnet Shard 0: `1666600000`
-  * Testnet Shard 0: `1666700000`
+  * Devnet Shard 0: `1666700000`
```
Comment (akegaviar) [inline docs/operations/harmony/tools.md:547]: `1666900000` see https://docs.harmony.one/home/general/wallets/browser-extensions-wallets/metamask-wallet/adding-harmony

File: docs/operations/harmony/tools.md [hunk 3]
```diff
@@ -596,7 +596,7 @@ where
 * ENDPOINT — your node WSS endpoint.
 * NETWORK_ID — Harmony network ID:
   * Mainnet Shard 0: `1666600000`
-  * Testnet Shard 0: `1666700000`
+  * Devnet Shard 0: `1666700000`
```
Comment (akegaviar) [inline docs/operations/harmony/tools.md:599]: `1666900000` see https://docs.harmony.one/home/general/wallets/browser-extensions-wallets/metamask-wallet/adding-harmony

File: docs/operations/harmony/tools.md [hunk 4]
```diff
@@ -641,7 +641,7 @@ where
 * ENDPOINT — your node HTTPS or WSS endpoint.
 * NETWORK_ID — Harmony network ID:
   * Mainnet Shard 0: `1666600000`
-  * Testnet Shard 0: `1666700000`
+  * Devnet Shard 0: `1666700000`
```
Comment (akegaviar) [inline docs/operations/harmony/tools.md:644]: `1666900000` see https://docs.harmony.one/home/general/wallets/browser-extensions-wallets/metamask-wallet/adding-harmony

File: docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md [hunk 5]
```diff
@@ -173,7 +173,7 @@ where
 
 * CONTRACT_NAME — the name of the contract as provided in the contract code `contract Polyland is ERC721, Ownable`.
 * CONTRACT_PATH — full path to the flattened contract.
-* PRIVATE_KEY — the private key to the account that deploys the contract. Must be used without the `0x` prefix. Fund the account with testnet ONE using the [testnet faucet](https://faucet.pops.one/).
+* PRIVATE_KEY — the private key to the account that deploys the contract. Must be used without the `0x` prefix. Fund the account with devnet ONE using the [devnet faucet](https://faucet.pops.one/).
```
Comment (akegaviar) [inline docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md:176]: http://dev.faucet.easynode.one/

File: docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md [hunk 6]
```diff
@@ -187,11 +187,11 @@ Once the contract deploys, note the `solc` and the `Deployed to` values in the o
 
 ### Verify the contract
 
-1. Open the [Harmony testnet explorer](https://explorer.pops.one/).
+1. Open the [Harmony devnet explorer](https://explorer.pops.one/).
```
Comment (akegaviar) [inline docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md:190]: https://explorer.ps.hmny.io/

File: docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md [hunk 7]
```diff
@@ -200,7 +200,7 @@ This will verify the contract. You can now use the explorer as a web app to inte
 
 ### Distribute the patches of land
 
-1. On the contract page in the [Harmony testnet explorer](https://explorer.pops.one/), click **Write Contract**
+1. On the contract page in the [Harmony devnet explorer](https://explorer.pops.one/), click **Write Contract**
```
Comment (akegaviar) [inline docs/tutorials/harmony/simple-metaverse-contract-with-foundry.md:203]: https://explorer.ps.hmny.io/

**Review by akegaviar — CHANGES_REQUESTED** [review]
Please see comments

**Comment by GranderStark** [comment]
And at the same time same variables were set in UI correctly in the first place. T-T 
Thanks, @akegaviar 
please re-review

**Review by akegaviar — APPROVED** [review]

**Pull request merged**